roles/qemu-kvm-ev/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: ensure yum vars contentdir is present with proper value
+  lineinfile:
+    path: /etc/yum/vars/contentdir
+    line: centos
```

Comment:
will this be OK if the line is appended to the file?

As I understand it, this file needs one value in it. If the file exists with an existing value, this may cause problems.